llvm/test/Bitcode/compatibility.ll

```diff
@@ -1122,6 +1122,26 @@ define void @fastMathFlagsForArrayCalls([2 x float] %f, [2 x double] %d1, [2 x <
   ret void
 }
 
+declare { float, float } @fmf_struct_f32()
+declare { double, double } @fmf_struct_f64()
+declare { <4 x double>, <4 x double> } @fmf_struct_v4f64()
```

Comment:
Test some cases with more than 2 entries

Reply:
And empty and an odd number of entries

Reply:
And complex nested structs

Reply:
Nested structs and empty structs are not supported (so can't have the attributes in the IR without an error), so I'll instead add those tests to the `InstructionsTest.cpp` unittest